### PR TITLE
Feat/make launchers

### PR DESCRIPTION
**Pull request by linuxbandit**
still gotta test all the commands

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,14 +17,54 @@ start: #docker-compose up -d
...
+clean_docker_dangling_images:
+	docker rmi $(docker images -qf "dangling=true")
+
+clean_docker_images:
+	docker rmi $(docker images -q)
+
+clean: clean_docker_images clean_docker_dangling_images
+
+prune: clean
+	docker system prune -y
```
Comment (serge1peshcoff) [inline Makefile]: `-a` maybe? this is what I do all the time

Comment (linuxbandit) [reply Makefile]: that's not a good command imho, "too much" cleaning

Comment (linuxbandit) [reply Makefile]: you do it only because we are forced to, being desperate for disk space

**Review by serge1peshcoff — APPROVED** [review]

**Comment by linuxbandit** [comment]
tested commands

**Pull request merged**